runtime/parachains/src/dmp.rs

```diff
@@ -14,13 +14,34 @@
 // You should have received a copy of the GNU General Public License
 // along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
 
+//! To prevent Out of Memory errors on the `DownwardMessageQueue`,
```

Comment:
formatting and line wrapping in this doc comment is somewhat not on par with the rest of the code; always been suggesting using https://marketplace.visualstudio.com/items?itemName=stkb.rewrap or similar.

Reply:
Some spacing would also help; see [this line](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) from this PR as an inspiration.